### DISCUSSION TITLE
what are the different layers for the sprites?

### DISCUSSION THREAD

**Discussion by Twelffth** [Monika-After-Story/MonikaModDev#9246] [QA - tech support + general] [answered]
In some of the code I see layer 4,2 etc. but which is which like the very top and very bottom

**Comment by multimokia** [comment, marked as answer]
A render order is described on our [wiki](https://github.com/Monika-After-Story/MonikaModDev/wiki/Official-MAS-Art-PSDs-and-Guidelines#render-order)

**Reply by Twelffth** [reply]
tysm!
